Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -325,7 +331,7 @@ protected class ClassWriter : IClassWriter {
 
     protected override void EmitLiteralExpr(ConcreteSyntaxTree wr, LiteralExpr e) {
       if (e.Value is bool value) {
-        wr.Write(value ? "True" : "False");
+        wr.Write("{0}", value);
```

Comment:
Interesting. I did not know `String.format("{0}", true)` would return `"True"` in csharp.